### DISCUSSION TITLE
Is Tasmocompiler broken?

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#18544] [Q&A]
Hello, I am trying to launch online TasmoCompiler, however it is not working as it is used to. Now the browser does not give the pop up prompt. Neither does http://localhost:3000/ link work and at bottom right of browser it says "No Open Ports". Please see the attached image:

![Screenshot (210)](https://user-images.githubusercontent.com/69470381/235354637-703b50f9-7ce8-4af3-828a-6d0ed6583e85.png)

Is there a problem with Online Tasmocompiler or is this a problem on my local computer. Please help. I have also posted https://github.com/benzino77/tasmocompiler/discussions/352 in TasmoCompiler discussion board.

**Comment by Jason2866** [comment]
Ask as TasmoCompiler Github. TasmoCompiler is not maintained from us.
